.goreleaser.yml

```diff
@@ -0,0 +1,69 @@
+project_name: newrelic-cli
+
+env:
+  - GO111MODULE=on
+
+before:
+  hooks:
+    - make clean
+    - go mod download
+
+builds:
+  -
+    id: newrelic
+    dir: cmd/newrelic
+    binary: newrelic
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+    ldflags:
+      - -s -w -X main.version={{.Version}} -X main.AppName={{.Binary}}
```

Comment:
Are the flags here intended to be on the same line?

Reply:
They do not have to be.  I can move it up, but makes for a long line and this works correctly #yaml

Reply:
Two lines is fine, just surprised that it works.

Reply:
YAML is weird, but it's the indention that makes it work...